src/onyx/log/zookeeper.clj

```diff
@@ -151,15 +155,31 @@
       (do (Thread/sleep 500)
           (recur)))))
 
+(defn find-messaging [log]
```

Comment:
We should come up with a way to abort the subscription if this, or the job scheduler, can't be found and the user doesn't want to wait.